### ISSUE TITLE
Add maplint as an overlay layer

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#484]
**[Submitted to the original trac issue database at 8.22am, Thursday, 24th May 2007]**

Maplint is now rendered as transparent pngs (single color transparency, not alpha, should work in IE, too) so it would make sense to enable maplint as an overlay for the slippy map.

**Comment by tomhughes** [comment]
**Author: bob@cakebox.net**
**[Added to the original trac issue at 8.27am, Thursday, 24th May 2007]**

The following code sets up the overlay in openlayers:


```
  layerMaplint = new OpenLayers.Layer.TMS(
  "tiles@home (Maplint annotation)",
  "http://dev.openstreetmap.org/~ojw/Tiles/maplint.php/",
  {type:'png', getURL: get_osm_url, isBaseLayer: false} );
  layerMaplint.setVisibility(false);
  map.addLayer(layerMaplint);
```

**Comment by tomhughes** [comment]
**Author: 80n80n@gmail.com**
**[Added to the original trac issue at 8.36pm, Thursday, 24th May 2007]**

Done for www.informationfreeway.org

**Comment by tomhughes** [comment]
**Author: bob@cakebox.net**
**[Added to the original trac issue at 5.21pm, Tuesday, 29th May 2007]**

Patch suitable for the main openstreetmap.org site:


```
--- app/views/site/index.rhtml  (revision 3055)
+++ app/views/site/index.rhtml  (working copy)
@@ -82,6 +82,9 @@
     map.addLayer(layer);
     layer = new OpenLayers.Layer.LikeGoogle( "Osmarender", "http://dev.openstreetmap.org/~ojw/Tiles/tile.php/", {type:'png'} );
     map.addLayer(layer);
+    layer = new OpenLayers.Layer.TMS( "Maplint", "http://dev.openstreetmap.org/~ojw/Tiles/maplint.php/", {type:'png', getURL: get_osm_url, isBaseLayer: false} );
+    layer.setVisibility(false); // Maplint should not be visible by default
+    map.addLayer(layer);
 
     //Marker stuff
```

**Comment by tomhughes** [comment]
**Author: tom@compton.nu**
**[Added to the original trac issue at 12.47pm, Sunday, 19th August 2007]**

The problem is the maplint layer (or at least large parts of it) do not actually appear to be transparent... Pass the ticket back to me when that is sorted and I can fix the slippy map to include it.

**Comment by tomhughes** [comment]
**Author: bob@cakebox.net**
**[Added to the original trac issue at 1.36am, Wednesday, 26th December 2007]**

Replying to [comment:4 tom@compton.nu]:
> The problem is the maplint layer (or at least large parts of it) do not actually appear to be transparent... Pass the ticket back to me when that is sorted and I can fix the slippy map to include it.

As said in #612 this looks good now.

**Comment by tomhughes** [comment]
**[Added to the original trac issue at 9.08am, Monday, 31st December 2007]**

(In [6196]) Add a maplint layer to the map. Note that the t@h server currently serves
solid grey tiles for tiles which are not in the database which limits the
utility of this.

Closes #484.
